index.html

```diff
@@ -8,16 +8,16 @@
         <meta name="description" content="Veteran-founded and led, Operation Code is a 501(c)(3) non-profit whose mission is to aid military, vets and their families learn coding and web technologies.">
         <meta name="keywords" content="operation code, veterans, software development, learn to code, keep coding, veteran coders, senate veterans, house veterans, gi bill, new gi bill, operationcode.org, #operationcode, military vets, vets">
         <meta name="author" content="Veteran Coders, for Veterans Coders">
-        <meta property="og:image" content="https://static1.squarespace.com/static/586d87de725e2585df6e8448/t/5870110d6b8f5b61fdd8ee05/1483739408996/?format=500w" />
-
+        <meta property="og:image" content="/static/media/Medal-Icon.0de4b44b.svg" />
         <link rel="stylesheet" type="styles/css" href="styles/css/styles.css">
-        <script src="https://cdn.ravenjs.com/3.16.0/raven.min.js" crossorigin="anonymous"></script>
-        <script type="text/javascript"> window.heap=window.heap||[],heap.load=function(e,t){window.heap.appid=e,window.heap.config=t=t||{};var r=t.forceSSL||"https:"===document.location.protocol,a=document.createElement("script");a.type="text/javascript",a.async=!0,a.src=(r?"https:":"http:")+"//cdn.heapanalytics.com/js/heap-"+e+".js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(a,n);for(var o=function(e){return function(){heap.push([e].concat(Array.prototype.slice.call(arguments,0)))}},p=["addEventProperties","addUserProperties","clearEventProperties","identify","removeEventProperty","setEventProperties","track","unsetEventProperty"],c=0;c<p.length;c++)heap[p[c]]=o(p[c])}; heap.load("405779510"); </script>
     </head>
 
     <body>
         <div id="app"></div>
-        <script>Raven.config('https://23e9c0c37a3841b8a692c98b8978f515@sentry.io/147247').install();</script>
         <script src="scripts/src/app.js"></script>
+        <script>Raven.config('https://23e9c0c37a3841b8a692c98b8978f515@sentry.io/147247').install();</script>
```

Comment:
What's the reasoning behind moving this? According to the [docs](https://docs.sentry.io/clients/javascript/):
> This script tag should be included after other libraries are loaded, but before your main application code (e.g. app.js):